skype/skype-ps/skype/Set-CsUserTeamMembers.md

```diff
@@ -82,7 +82,7 @@ Accept wildcard characters: False
 ```
 
 ### -User
-{{ Fill User Description }}
+Indicates the Identity of the user account to be modified. User Identities can be specified using one of four formats: 1) the user's SIP address; 2) the user's user principal name (UPN); 3) the user's domain name and logon name, in the form domain\logon (for example, litwareinc\kenmyer) and 4) the user's Active Directory display name (for example, Ken Myer). User Identities can also be referenced by using the user's Active Directory distinguished name.
```

Comment:
I think you can drop "user's" from each of the four formats., i.e. "the user principal name (UPN)".